discovery/seata-discovery-raft/src/main/java/org/apache/seata/discovery/registry/raft/RaftRegistryServiceImpl.java

```diff
@@ -336,22 +338,41 @@ private static boolean watch() throws RetryableException {
     }
 
     @Override
-    public List<InetSocketAddress> refreshAliveLookup(String transactionServiceGroup,
-        List<InetSocketAddress> aliveAddress) {
+    public List<InetSocketAddress> refreshAliveLookup(String transactionServiceGroup, List<InetSocketAddress> aliveAddress) {
```

Comment:
Hi @SAIVARDHAN15 , when refreshAliveLookup is called, it has already undergone a health check by NettyClientChannelManager#doReconnect. I tested it locally and confirmed that the health check is normal, so this task might be a misunderstanding? Due to the issues from #6919 and #6917, I have been mistakenly thinking that the health check was failing. If you're interested, you can take on these two tasks.